### PR TITLE
Adding OData processor to readme.

### PR DESCRIPTION
**Pull request by EricAtMSFT**
Adding a snippet about the shiny new OData processor. 🥇

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,6 +20,8 @@ The base JSON and YAML processors are built into this project. Below is the list
 
 - [**C# Comment / Annotation Processor**](https://github.com/Microsoft/OpenAPI.NET.CSharpAnnotations) : Converts standard .NET annotations ( /// comments ) emitted from your build (MSBuild.exe) into OpenAPI.NET document object. 
 
+- [**OData CSDL Processor**](https://github.com/Microsoft/OpenAPI.NET.OData) : Converts the XML represetntation of the Entity Data Model (EDM) into OpenAPI.NET document object.
```
Comment (PerthCharern) [inline README.md]: representation typo

Comment (PerthCharern) [reply README.md]: looks like that got carried over from the OData repo :)

Comment (PerthCharern) [inline README.md]: EDM is a generic term that also applies to non-OData stuff. Is it better to rephrase this to the one below?

Converts the XML representation of the Entity Data Model (EDM) describing an OData Service into OpenAPI.NET document object.

**Comment by EricAtMSFT** [comment]
Fixed.
